jsx/Form.js

```diff
@@ -255,6 +255,197 @@ var SelectElement = React.createClass({
   }
 });
 
+var ListElement = React.createClass({
```

Comment:
I just looked at the screenshot, and I don't think "List" is the right word for this element type, but I'm drawing a blank for what they're called. (Tags, maybe? What do bootstrap or other UI frameworks call them?)

Reply:
yeah, Tags seems to be the common name for this sort of thing. will go with that